styles/index.css

```
@@ -0,0 +1,109 @@
+body {
+    /* background-color: rgb(91, 133, 229); */
+    background-image: url('../assets/weather.jpg');
+}
+
+main {
+    font-family: cursive;
+    font-size: larger;
+    font-weight: bolder;
+    text-align: center;
+}
+
+main h1 {
+    margin-bottom: 0px;
+}
+
+.top-city-title {
+    display: inline-block;
+    white-space: nowrap;
+}
+
+.white-box {
+    font-family: 'Courier New', Courier, monospace;
+    font-weight: bolder;
+    color:rgb(3, 2, 3);
+    background-position: center;
+    background-size: cover;
+    text-align: center;
+    background: rgb(19, 53, 79);
+    background-color: rgb(213, 140, 233);
+    margin-left: 600px;
+    margin-right: 600px;
+    margin-top: 50px;
+    padding-bottom: 5px;
+    border-radius: 50px;
+    opacity: 0.8;
+}
+
+#up-button {
+    background-color: aqua;
+    border-radius: 100px;
+}
+
+#down-button {
+    background-color: aqua;
+    margin-bottom: 10px;
+    border-radius: 100px;
+}
+
+#realtimeTemp {
+    padding: 10px;
+    border-radius: 50px;
+    background-color: whitesmoke;
+    border-style: dotted;
+}
+
+#sky-button {
+    padding: 10px;
+    border-radius: 50px;
+    background-color: whitesmoke;
+    border-style: dotted;
+}
+
+#reset-button {
+    padding: 10px;
+    border-radius: 50px;
+    background-color: whitesmoke;
+    border-style: dotted;
+}
+
+.teal {
```

These classes mention the colors but not what they mean.  If you wanted to change the color scheme (say red to purple to blue) then you would need to rename all of these.

It would be better if this was named something like `cold`:

```suggestion
.cold {
```